### DISCUSSION TITLE
Internal Error :: Sending Email

### DISCUSSION THREAD

**Discussion by techszip** [postalserver/postal#2493] [Help with using Postal]
Hello,

Getting error while sending email 
---------------------------------------------
An internal error occurred while sending 
this message. This message will be retried automatically. If this 
persists, contact support for assistance.

FrozenError: can't modify frozen String: ""
Support Ref: J-KZBSWBOLKC

Thank you

**Comment by willpower232** [comment]
Have a look at https://github.com/postalserver/postal/issues/2153

Unfortunately there hasn't been a new release tagged yet to provide the fix.

**Comment by gigbuilder** [comment]
Same problem here -- I wiped the server, reinstalled everything from scratch, ran for about 3 days, then this frozen error returned.
